src/Microsoft.ServiceFabric.Actors/Remoting/V2/Runtime/ActorServiceRemotingDispatcher.cs

```diff
@@ -26,8 +26,8 @@ namespace Microsoft.ServiceFabric.Actors.Remoting.V2.Runtime
     public class ActorServiceRemotingDispatcher : ServiceRemotingMessageDispatcher
     {
         private static readonly string TraceType = typeof(ActorServiceRemotingDispatcher).Name;
-        private readonly ActorService actorService;
-        private readonly ServiceRemotingCancellationHelper cancellationHelper;
+        private ActorService actorService;
```

Comment:
Why are we making these fields writeable? If this is for tests, then readonly instance fields can be modified by reflection, only readonly static fields cannot be.